### DISCUSSION TITLE
AttributeError: Can't get attribute '_rebuild_function'

### DISCUSSION THREAD

**Discussion by Keamww2021** [MaartenGr/BERTopic#1248] [Q&A]
helllo

i need help to solve this problem.
i was working perfectly on bertopic version 12
then, i decieded to upgrade bertopic to 14
when i tray to load my brtopic model i could not
so, i create new environment with bertopic version 12 then i tries to load the model
i get this error:

AttributeError: Can't get attribute '_rebuild_function' on <module 'numba.core.serialize' from 'C:\Users\96659\anaconda3\envs\oldBert\lib\site-packages\numba\core\serialize.py'>

can someone help

**Comment by MaartenGr** [comment]
Closing this in favor of #1247
